example/package.json

```diff
@@ -31,27 +31,28 @@
     "prop-types": "^15.6.2",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
-    "react-relay": "^6.0.0",
-    "relay-runtime": "^6.0.0",
+    "react-relay": "^7.0.0",
+    "relay-runtime": "^7.0.0",
     "todomvc-app-css": "^2.1.0",
     "todomvc-common": "^1.0.3",
     "webpack": "^3.5.5",
     "webpack-dev-server": "^2.7.1",
     "whatwg-fetch": "2.0.3"
   },
   "devDependencies": {
+    "@types/node": "^12.11.7",
```

Comment:
Why is this needed?

Reply:
When not installed, `yarn start` on example folder gives the following error:
```bash
ERROR in /Users/renan/www/relay-compiler-language-typescript/example/node_modules/@types/node/index.d.ts(200,11):
TS2300: Duplicate identifier 'IteratorResult'.
ERROR in /Users/renan/www/relay-compiler-language-typescript/example/node_modules/typescript/lib/lib.es2015.iterable.d.ts(41,6):
TS2300: Duplicate identifier 'IteratorResult'.
```

Solved upgrading `@types/node` suggested [here](https://github.com/microsoft/TypeScript/issues/32333#issuecomment-522408244).

Reply:
Ok, so somehow we were already pulling in this package anyways, just an older version. Cool thanks 👍